src/Acrobot.jl

```diff
@@ -116,48 +122,47 @@ function _dsdt(env::AcrobotEnv, s_augmented, t)
     I1 = env.LINK_MOI
```

Comment:
This function calculates the gradients for the Runge-Kutta. Is there a way we can use Flux here?